typescript/infra/scripts/check/check-owner-ica.ts

```diff
@@ -29,17 +32,32 @@ async function main() {
   const { chainAddresses } = await getHyperlaneCore(environment, multiProvider);
   const ica = InterchainAccount.fromAddressesMap(chainAddresses, multiProvider);
 
+  const checkOwnerIcaChains = (
+    chains?.length ? chains : Object.keys(icas)
+  ).filter(isEthereumProtocolChain);
+
   const ownerConfig: AccountConfig = {
     origin: ownerChain,
     owner: owner,
   };
+  const ownerChainInterchainAccountRouter =
```

Comment:
should there also be a zero check here? just in case

Reply:
hopefully never hit it, but yes i'll add it